daphne_worker/src/tracing_utils.rs

```diff
@@ -162,13 +162,35 @@ pub fn initialize_tracing(env: &Env) {
             Ok(var) => var.to_string(),
             Err(_) => "info".to_string(),
         };
-        registry()
-            .with(
-                fmt::layer()
+
+        let (ansi, json) = match env.var("DAP_TRACING_FORMAT") {
```

Comment:
Instead of adding another environment variable, I would prefer using `DAP_DEPLOYMENT` to control this behavior:
`dev` -> console print
`prod` -> JSON